options.html

```diff
@@ -1,162 +1,128 @@
+            <div class="custom-control custom-radio">
+                <input type="radio" id="toneColorsNone" name="toneColors" class="custom-control-input">
+                <label class="custom-control-label" for="toneColorsNone">Don't show tone colors.</label>
+            </div>
+        </div>
+
+        <div class="form-group mb-4">
+            <label>Font size of characters in the pop-up window.</label>
+            <div class="custom-control custom-radio">
+                <input type="radio" id="fontSizeSmall" name="fontSize" class="custom-control-input">
+                <label class="custom-control-label" for="fontSizeSmall">Display characters in a smaller font size.</label>
+            </div>
+            <div class="custom-control custom-radio">
+                <input type="radio" id="fontSizeLarge" name="fontSize" class="custom-control-input">
+                <label class="custom-control-label" for="fontSizeLarge">Display characters in a larger font size.</label>
+            </div>
+        </div>
+
+        <div class="form-group mb-4">
+            <label>Character style in the pop-up window.</label>
+            <div class="custom-control custom-radio">
+                <input type="radio" id="simpTradClassic" name="simpTrad" class="custom-control-input">
+                <label class="custom-control-label" for="simpTradClassic">Display both simplified and traditional characters.</label>
+            </div>
+            <div class="custom-control custom-radio">
+                <input type="radio" id="simpTradAuto" name="simpTrad" class="custom-control-input">
+                <label class="custom-control-label" for="simpTradAuto">Display character style automatically detected on the page.</label>
+            </div>
+        </div>
+
+        <div class="form-group mb-4">
+            <label>Additional Chinese transliterations.</label>
+            <div class="custom-control custom-checkbox">
+                <input type="checkbox" class="custom-control-input" id="zhuyin">
+                <label class="custom-control-label" for="zhuyin">Show <a href="https://wikipedia.org/wiki/Bopomofo" target="_blank">Zhuyin (Bopomofo)</a> phonetic symbols.</label>
+            </div>
+        </div>
+
+        <div class="form-group mb-4">
+            <label>Grammar and usage notes from the <a href="https://resources.allsetlearning.com/chinese/grammar" target="_blank">Chinese Grammar Wiki</a>.</label>
+            <div class="custom-control custom-checkbox">
+                <input type="checkbox" class="custom-control-input" id="grammar">
+                <label class="custom-control-label" for="grammar">Show grammar and usage notes.</label>
+            </div>
+        </div>
+        <h2 class="mb-3">Other Options</h2>
+
+        <div class="form-group mb-4">
+            <label>Save entries to <a href="https://github.com/cshiller/zhongwen/tree/readme#built-in-word-list" target="_blank">Built-in Word List</a>.</label>
```

Comment:
Link is broken, href should be set to https://github.com/cschiller/zhongwen/#built-in-word-list

Reply:
I'll fix it.

Regarding that link, I was also wondering whether to link the Readme or open the actual Word List. (I chose the first one because it seemed easier.)

Reply:
I think the Readme you picked is fine. If someone doesn't know about this feature it would otherwise only take them to an empty list.